lib/applications/schema.ts

```diff
@@ -376,8 +377,10 @@ export default gql`
 
     # Payment information (omit processing fee)
     paymentMethod: PaymentType!
-    # Input monetary value as string
-    donationAmount: String
+    donationAmount: String # Input monetary value as string
+    paidThroughShopify: Boolean!
+    shopifyPaymentStatus: ShopifyPaymentStatus
+    shopifyConfirmationNumber: String
```

Comment:
Is there a reason for exposing these fields to the API? I kept them hidden because only renewal applications can currently be paid for via Shopify, but I want to keep the Shopify open for new/replacement applications in the future as well. New applications shouldn't have this exposed, as they must be paid for via mail-in or in-office, unless I'm missing something

Reply:
Oh my bad, I'll remove these. Didn't realize new applications can't pay with shopify rn